hydra-node/test/Hydra/Chain/Direct/Contract/Close.hs

```diff
@@ -254,13 +304,13 @@ genCloseMutation (tx, _utxo) =
   oneof
     [ SomeMutation (Just $ toErrorCode InvalidSnapshotSignature) MutateSignatureButNotSnapshotNumber . ChangeHeadRedeemer <$> do
         Head.Close . toPlutusSignatures <$> (arbitrary :: Gen (MultiSignature (Snapshot Tx)))
-    , SomeMutation (Just $ toErrorCode ClosedWithNonInitialHash) MutateSnapshotNumberToLessThanZero <$> do
+    , SomeMutation (Just $ toErrorCode ClosedWithNonInitialHash) MutateSnapshotNumberToLessThanEqualZero <$> do
         mutatedSnapshotNumber <- arbitrary `suchThat` (<= 0)
         pure $ ChangeOutput 0 $ changeHeadOutputDatum (replaceSnapshotNumber mutatedSnapshotNumber) headTxOut
     , SomeMutation (Just $ toErrorCode InvalidSnapshotSignature) MutateSnapshotNumberButNotSignature <$> do
-        mutatedSnapshotNumber <- arbitrarySizedNatural `suchThat` (\n -> n /= healthySnapshotNumber && n > 0)
+        mutatedSnapshotNumber <- arbitrarySizedNatural `suchThat` (> healthyCloseSnapshotNumber)
```

Comment:
I've found this was changed in commit [3523b962f5](https://github.com/input-output-hk/hydra/commit/3523b962f505ccc5cbeeb0e2aece26c0c2296666):

> Simplify arbitrary condition to also be consistent with similar mutation for contest

I would add some explanations to make it clear why only checking it's bigger than `healthyCloseSnapshotNumber` is equivalent to `n /= healthySnapshotNumber && n > 0`

Reply:
I did it so it is consistent with Contest mutations.

It is not fully equivalent, as there are missing values not being tested btw 0 and healthySnapshotNumber but I do believe it is enough to test bigger snapshots than the healthy in order to check this property.

Hopefully this answer your question :)